### PR TITLE
Provide Solr's LOGO

### PR DESCRIPTION
**Pull request by dmsolr**
image

### CODE REVIEW

**Comment by wu-sheng** [comment]
Learn from this PR, https://github.com/apache/skywalking-rocketbot-ui/pull/113

File: src/views/components/topology/topo.vue [hunk 1]
```diff
@@ -305,7 +306,7 @@ export default {
   },
     toggleNode(nodeCircle, currNode, isHover) {
     if (isHover) {
-        // 提升节点层级 
+        // ??????????????????
```
Comment (wu-sheng) [inline src/views/components/topology/topo.vue]: Remove question mark and Chinese comments.

Comment (dmsolr) [reply src/views/components/topology/topo.vue]: Sorry, My editor dosen't  set encoding. 
I remove directly the comments?

Comment (wu-sheng) [reply src/views/components/topology/topo.vue]: Yes.

**Pull request merged**